python/smashbox/utilities/__init__.py

```diff
@@ -570,7 +570,7 @@ def share_file_with_user(filename, sharer, sharee, **kwargs):
         # TODO: this code is not the best - the goal is to trap a share not allowed error and return that error code
 
         logger.info('Share failed with %s', str(err))
-        if "not allowed to share" in str(err):
+        if 'HTTP error: 403' == err.message:
```

Comment:
Back then I didn't know how to, see https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L22
Needs further research. Feel free to do it yourself (am busy with locking stuff).
If not, then raise a ticket there for it to be added later.